### ISSUE TITLE
AUR package failed to build

### ISSUE THREAD

**Issue by issadarkthing** [tramhao/termusic#82]
The AUR package failed to build

## Error log
```
==> Making package: termusic 0.6.16-1 (Ahad 03 Jul 2022 03:18:48 )
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Downloading termusic-0.6.16.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 3188k    0 3188k    0     0  2905k      0 --:--:--  0:00:01 --:--:-- 18.9M
==> Validating source files with sha512sums...
    termusic-0.6.16.tar.gz ... Passed
==> Extracting sources...
  -> Extracting termusic-0.6.16.tar.gz with bsdtar
==> Starting prepare()...
error: failed to parse manifest at `/tmp/termusic-8379323242555384452/termusic/src/termusic-0.6.16/Cargo.toml`

Caused by:
  namespaced features with the `dep:` prefix are only allowed on the nightly channel and requires the `-Z namespaced-features` flag on the command-line
==> ERROR: A failure occurred in prepare().
    Aborting...
```

## Machine spec

```
OS: Artix Linux x86_64
Host: VivoBook_ASUSLaptop X513EPN_A513EP 1.0
Kernel: 5.18.6-artix1-1
Uptime: 7 hours, 6 mins
Packages: 1365 (pacman)
Shell: zsh 5.9
Resolution: 1920x1080
WM: xmonad
Theme: Artix-dark [GTK2/3]
Icons: matefaenzadark [GTK2/3]
Terminal: alacritty
CPU: 11th Gen Intel i5-1135G7 (8) @ 4.200GHz
GPU: Intel TigerLake-LP GT2 [Iris Xe Graphics]
GPU: NVIDIA GeForce MX330```

**Comment by tramhao** [comment]
Today I merged v0.6.17 and rust MSRV is v1.57.0.

**Comment by issadarkthing** [comment]
It compiles now, thanks!
